python/nav/web/api/v1/views.py

```diff
@@ -259,6 +289,10 @@ class NetboxViewSet(NAVAPIMixin, viewsets.ReadOnlyModelViewSet):
     filter_fields = ('ip', 'sysname', 'room', 'organization', 'category')
     search_fields = ('sysname', )
 
+    def pre_delete(self, obj):
```

Comment:
maybe make a note of the fact that `pre_delete()` is no longer supported on Django REST-framework 3.0